lib/logstash/inputs/rabbitmq.rb

```diff
@@ -52,6 +52,10 @@ class RabbitMQ < LogStash::Inputs::Threadable
       # * Wildcards are not valid on direct exchanges.
       config :key, :validate => :string, :default => "logstash"
 
+      # Amount of time in seconds to wait after a failed subscription request
+      # before retrying. Subscribes can fail if the server goes away and then comes back
+      config :subscription_retry_interval_seconds, :validate => :number, :required => true, :default => 5
```

Comment:
In case we ever allow people to set a `null` value. Future proofing :) ? :) :smiling_imp: